src/assignments/6-array.js

```diff
@@ -21,3 +31,8 @@ min=1
 
 */
 // start here writing the code for program2
+const arr = [12,56,78,45];
+const max = Math.max(...arr);
+const min = Math.min(...arr);
+console.log(`max = ${max}`);
```

Comment:
write the function to find out max and min in the array, not to use the max min written by javascript library. 
You can get the logic from here:
(https://www.w3resource.com/c-programming-exercises/array/c-array-exercise-9.php)